R/series-s3-base.R

```diff
@@ -67,3 +67,18 @@ as.vector.polars_series <- function(x, mode = "any") {
 
 #' @export
 length.polars_series <- function(x) x$len()
+
+#' @export
+max.polars_series <- function(x, ...) x$max()
+
+#' @export
+min.polars_series <- function(x, ...) x$min()
+
+#' @export
+mean.polars_series <- function(x, ...) x$mean()
+
+#' @exportS3Method stats::median
+median.polars_series <- function(x, ...) x$median()
```

Comment:
Maybe this function should be moved to `series-s3-stats.R`?

Reply:
It's part of base R so I think it's fine here, just like we don't have `series-s3-utils.R` for the dollar method.